src/promptflow/promptflow/core/_flow.py

```diff
@@ -127,13 +127,11 @@ def _resolve_cls_and_type(cls, data, params_override):
 
 
 class Flow(FlowBase):
-    """This class is used to hide internal interfaces from user.
+    """This class is public Flow interface.
```

Comment:
refine this line to sth like below:

A flow is ...

Reply:
https://microsoft.github.io/promptflow/concepts/concept-flows.html ?